support-workers/src/typescript/lambdas/createPaymentMethodLambda.ts

```diff
@@ -146,19 +165,44 @@ async function createStripeHostedPaymentMethod(
 		),
 		"Couldn't retrieve payment method ID from Stripe",
 	);
-	const customer = await stripeService.createCustomer(
+	return createStripePaymentMethod(
+		stripeService,
 		stripePublicKey,
 		paymentMethodId,
+		undefined,
 	);
+}
 
+async function createStripePaymentMethod(
+	stripeService: StripeService,
+	stripePublicKey: string,
+	paymentMethodId: string,
+	stripePaymentType: StripePaymentType | undefined,
+): Promise<StripePaymentMethod> {
+	const customer = await stripeService.createCustomer(
+		stripePublicKey,
+		paymentMethodId,
+	);
+	const stripePaymentMethod = await stripeService.getPaymentMethod(
+		stripePublicKey,
+		paymentMethodId,
+	);
+	const card = getIfDefined(
+		stripePaymentMethod.card,
+		'Card details not found on Stripe payment method',
+	);
 	return {
 		TokenId: paymentMethodId,
 		SecondTokenId: customer.id,
+		CreditCardNumber: card.last4,
+		CreditCardExpirationMonth: card.exp_month,
+		CreditCardExpirationYear: card.exp_year,
+		CreditCardType: zuoraCardTypeFromStripe(card.brand),
 		PaymentGateway: stripeService.getPaymentGateway(stripePublicKey),
 		Type: 'CreditCardReferenceTransaction',
+		StripePaymentType: stripePaymentType,
```

Comment:
apple and google pay must have been appearing in bigquery as normal stripe
https://github.com/guardian/support-frontend/blob/ce10d090f263960bee243755063016f50fcc7cca/support-workers/src/main/scala/com/gu/acquisitions/AcquisitionDataRowBuilder.scala#L112-L114

Reply:
not sure if the acquisition event bus data on stripe payment types is actually used for anything important though